### ISSUE TITLE
WLED controller support for Reef-pi

### ISSUE THREAD

**Issue by jrsphoto** [reef-pi/reef-pi#2054]
]Feature Request]
I'm looking for some cheeping RGB/RGBW lighting options and there just isn't a ton of options, unless you're willing to "roll your own".  [WLED ](https://kno.wled.ge/)is pretty much the go-to for RGB-led light controlling and supports a dizzying array of  LED strips and lighting effects.  It has both JSON and HTTP Request API's, and a ton of other features that could all be controlled from a page on the Reef-Pi UI.  This would really be a nice feature and enable some pretty cool lighting effects, in a very cost-effective way. 

Very basically, you need a WLED controller.  Something like [this](https://www.athom.tech/blank-1/wled-high-power-led-strip-controller), but others are listed on the WLED page above.  Most are ESP8266/ESP32 based.  Your LED PSU and LED strips connect to this device.  This device connects to your local wifi network and is then available to accept control commands.

-John

**Comment by jrsphoto** [comment]
More info on the [HTTP ](https://kno.wled.ge/interfaces/http-api/)and [JSON ](https://kno.wled.ge/interfaces/json-api/)API:

**Comment by jrsphoto** [comment]
The main WLED controller I use is the [QuinLED Digi-Quad](https://quinled.info/pre-assembled-quinled-dig-quad/), which is a 4-channel controller.  There is a [video of the board designer](https://www.youtube.com/watch?v=JIkoUCuXFa0&ab_channel=Intermit.Tech) showing how the board works..

**Comment by ranjib** [comment]
Yes , I like this one too. @jrsphoto any help with coding/driver development? well need a driver for this one

**Comment by robertmosen** [comment]
I would like this too.
